ovn-tester/ovn_workload.py

```diff
@@ -140,21 +142,35 @@ def configure_localnet(self, physical_net):
         self.run(cmd=f'ovs-vsctl -- set open_vswitch . '
                  f'external-ids:ovn-bridge-mappings={physical_net}:br-ex')
 
+    def configure_external_host_net_str(self, ip, gw):
+        cmd = ''
+        if ip.get_ip4():
+            cmd += f'ip -n ext-ns addr add {ip.get_ip4()}/{ip.get_plen4()}' \
```

Comment:
Nit: While 'ip -n <ns> <cmd>' is shorter, until now we always used the expanded version:
```
ip netns exec ext-ns ip addr add ...
```
I think we should maintain consistency and use only one of the command forms.  This applies to multiple places where `ip -n ... ` is used.

Reply:
@dceara ack, I will fix it